packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -985,10 +993,15 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
             mode: 'reveal'
         } : undefined;
 
-        const editorWidget = await this.editorManager.open(fileUri, opts);
+        if (preview) {
+            opts = Object.assign(opts || {}, { preview });
+        }
+
+        const opener = await this.openerService.getOpener(fileUri, opts);
+        const editorWidget = await opener.open(fileUri, opts) as EditorWidget;
```

Comment:
These can be replaced by a single call to the `open()` function available from `'@theia/core/lib/browser'`